addons_providers.go

```diff
@@ -15,24 +15,49 @@ type AddonProvidersService interface {
 
 var _ AddonProvidersService = (*Client)(nil)
 
+type Pricing struct {
+	Price    string `json:"price"`
+	Currency string `json:"currency"`
+	Period   string `json:"period"`
+}
+
 type Plan struct {
-	ID          string  `json:"id"`
-	LogoURL     string  `json:"logo_url"`
-	DisplayName string  `json:"display_name"`
-	Name        string  `json:"name"`
-	Description string  `json:"description"`
-	Price       float64 `json:"price"`
-	SKU         string  `json:"sku"`
+	ID                        string             `json:"id"`
+	DisplayName               string             `json:"display_name"`
+	Price                     float64            `json:"price"`
+	Name                      string             `json:"name"`
+	Description               string             `json:"description"`
+	Position                  int                `json:"position"`
+	OnDemand                  bool               `json:"on_demand"`
+	Disabled                  bool               `json:"disabled"`
+	DisabledAlternativePlanID bool               `json:"disabled_alternative_plan_id"`
+	SKU                       string             `json:"sku"`
+	Pricings                  map[string]Pricing `json:"pricings"`
+	HDSAvailable              bool               `json:"hds_available"`
 }
 
 type PlansParams struct {
 	Plans []*Plan `json:"plans"`
 }
 
+type Category struct {
+	ID          string `json:"id"`
+	Name        string `json:"name"`
+	Description string `json:"description"`
+	Position    int    `json:"position"`
+}
+
 type AddonProvider struct {
-	ID      string `json:"id"`
-	LogoURL string `json:"logo_url"`
-	Name    string `json:"name"`
+	ID               string    `json:"id"`
+	LogoURL          string    `json:"logo_url"`
+	Name             string    `json:"name"`
+	ShortDescription string    `json:"short_description"`
+	Description      string    `json:"description"`
+	Category         *Category `json:"category"`
+	ProviderName     string    `json:"provider_name"`
+	ProviderURL      string    `json:"provider_url"`
+	HDSAvailable     bool      `json:"hds_available"`
+	Plans            []*Plan   `json:"plans"`
```

Comment:
question: is there a good reason to use a pointer here? If not, let's not use a pointer
```suggestion
	Plans            []Plan   `json:"plans"`
```